tests/integration-tests/pom.xml

```diff
@@ -340,6 +402,19 @@
          </exclusions>
       </dependency>
 
+      <dependency>
+         <groupId>org.codehaus.woodstox</groupId>
+         <artifactId>woodstox-core-asl</artifactId>
+         <version>4.4.0</version>
```

Comment:
While we are 'cleaning up deps' and moving the dep declaration, lets at add a prop for the version as done for some others.